Mathlib/NumberTheory/LSeries/Convergence.lean

```diff
@@ -18,36 +18,36 @@ real part `x` and provide some results about it.
 L-series, abscissa of convergence
 -/
 
-namespace ArithmeticFunction
-
-open Complex Nat
+open Complex
 
 /-- The abscissa `x : EReal` of absolute convergence of the L-series associated to `f`:
 the series converges absolutely at `s` when `re s > x` and does not converge absolutely
 when `re s < x`. -/
-noncomputable def abscissaOfAbsConv (f : ArithmeticFunction ℂ) : EReal :=
+noncomputable def LSeries.abscissaOfAbsConv (f : ℕ → ℂ) : EReal :=
```

Comment:
Isn't it easier to define it by the infimum of sigma that `term f n n^(-sigma)` converges to 0, or is bounded, or even by the corresponding lim sup property . Compare with [FormalMultilinearSeries.radius](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Analytic/Basic.html#FormalMultilinearSeries.radius)

Reply:
It is not that simple; compare my comment on the indicator function of `k`th powers.
The convergence behavior of Dirichlet series is much more subtle than that of power series. E.g., there can be a difference of up to `1` between the abscissa of conditional convergence and that of absolute convergence.

Reply:
indeed, so forget about my remark.